### PR TITLE
Don't constexpr iphone_xs_times

### PR DESCRIPTION
**Pull request by gw280**
The clang toolchain for Fuchsia doesn't like this lambda capture to be explicit when `iphone_xs_times` is a constexpr.

### CODE REVIEW

**Comment by gw280** [comment]
This is a dupe of https://github.com/flutter/engine/pull/15450 but this fix should work in all cases. The fix in the other PR fails where the file currently compiles (as other compilers want the lambda to be explicitly captured)

File: shell/common/input_events_unittests.cc [hunk 1]
```diff
@@ -224,53 +224,56 @@ TEST_F(ShellTest, DelayAtMostOneEventForFasterThanVSyncInputEvents) {
...
-                                        41.1552099609375,
-                                        42.102128906249995,
-                                        43.0426318359375,
-                                        44.070131835937495,
-                                        45.08862304687499,
-                                        46.091469726562494};
+  //
+  // We don't use `constexpr double iphone_xs_times` here because Fuchsia's
+  // clang doesn't handle it well with lambda capture.
+  double iphone_xs_times[] = {0.15,
```
Comment (cbracken) [inline shell/common/input_events_unittests.cc]: Can this not be `const`?

Comment (gw280) [reply shell/common/input_events_unittests.cc]: Not sure. Let's see whether the compiler complains :)

Comment (gw280) [reply shell/common/input_events_unittests.cc]: seems fine, switched to const

**Comment by gw280** [comment]
An update, after talking with some compiler folks: Fuchsia's toolchain is much more up to date than e.g. the host toolchain on my Linux machine (11.x vs. 8.x). The compiler's behaviour here changed and the best workaround here is to use `static constexpr`.

**Pull request merged**